pandas/tests/io/test_parquet.py

```diff
@@ -238,6 +238,12 @@ def test_cross_engine_pa_fp(df_cross_compat, pa, fp):
 def test_cross_engine_fp_pa(df_cross_compat, pa, fp):
     # cross-compat with differing reading/writing engines
 
+    if pyarrow.__version__.startswith("0.14"):
+        pytest.xfail(
+            "Reading fastparquet with pyarrow in 0.14 fails: "
+            "https://issues.apache.org/jira/browse/ARROW-6492"
```

Comment:
In principle this also depends on the fastparquet version, but probably easier to just skip it altogether like you do. It might that this fails on 0.13 as well, not sure.

Reply:
It was failing for 0.13 too and I've added this to the XFail.